angio-frontend/src/modules/analyse/components/AnalyseListFilter.vue

```diff
@@ -0,0 +1,149 @@
+                                            v-bind:selected="data.selected"
+                                            v-bind:disabled="data.disabled"
+                                            class="v-chip--select-multi caption"
+                                            v-on:input="data.parent.selectItem(data.item)"
+                                            small
+                                    >
+                                        {{ $t(data.item.text) }}
+                                        <v-icon
+                                                small
+                                                v-on:click="data.parent.selectItem(data.item)"
+                                        >close</v-icon>
+                                    </v-chip>
+                                </template>
+                                <template v-slot:no-data>
+                                    <v-list-tile>
+                                        <v-list-tile-content>
+                                            <v-list-tile-title>
+                                                {{ $t('analyse.component.analyseListFilter.statusWithoutMatches') }}
+                                            </v-list-tile-title>
+                                        </v-list-tile-content>
+                                    </v-list-tile>
+                                </template>
+                            </v-combobox>
+                        </v-flex>
+                    </v-layout>
+                </v-flex>
+
+                <v-flex xs12>
+                    <v-layout align-center>
+                        <v-flex xs5>
+                            <PeriodDatePicker
+                                    v-bind:locale="locale"
+                                    v-bind:isPeriod="isPeriod"
+                                    v-on:send-period="onPeriodDataChanged"
+                            ></PeriodDatePicker>
+                        </v-flex>
+
+                        <v-flex xs5 pl-2>
+                            <v-checkbox
+                                    v-model="isPeriod"
+                                    v-bind:label="$t('analyse.component.analyseListFilter.period')"
+                            ></v-checkbox>
+                        </v-flex>
+
+                        <v-flex xs2 pl-2>
+                            <v-checkbox
+                                    v-model="isStarred"
+                                    v-bind:label="$t('analyse.component.analyseListFilter.starred')"
+                                    class="justify-end"
+                            ></v-checkbox>
+                        </v-flex>
+                    </v-layout>
+                </v-flex>
+            </v-layout>
+        </v-card-title>
+    </v-card>
+</template>
+
+<script lang="ts">
+    import {Component, Emit, Prop, Vue} from "vue-property-decorator";
```

Comment:
Only _single quote_ is used in all typescript files

Reply:
Done